geomesa-features/geomesa-feature-kryo/src/main/scala/org/locationtech/geomesa/features/kryo/KryoBufferSimpleFeature.scala

```diff
@@ -42,8 +42,9 @@ class KryoBufferSimpleFeature(sft: SimpleFeatureType,
   private var userDataOffset: Int = -1
 
   private var binaryTransform: () => Array[Byte] = input.getBuffer
+  private var reserializeTransform: () => Array[Byte] = input.getBuffer
```

Comment:
the functions get overwritten in the setTransform method - so they need to be vars.
Execution flow is:
val feature = // get reusable feature instance
feature.setTransform // set transforms once for this execution cycle
while (looping) { feature.setBuffer(newBytes); val result = feature.transform() }